core/tasks.py

```
@@ -256,3 +269,200 @@ def track_state_change(
+                ).order_by('created_at').first()
+                
+                if next_blog_post:
+                    # Schedule the blog post
+                    async_task(
+                        "core.tasks.submit_scheduled_blog_post",
+                        next_blog_post.id,
+                        schedule=next_post_time,
+                        group="Blog Post Submission"
+                    )
+                    
+                    result = f"Scheduled blog post '{next_blog_post.post_title}' for {project.name} at {next_post_time}"
```

Ruff (0.11.9)

Line too long (120 > 100)

(E501)